LifeHelper.Services/Areas/User/DTOs/UserRegisterDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace LifeHelper.Services.Areas.User.DTOs;
+
+public class UserRegisterDto
+{
```

Comment:
maybe, it will be pretty rename this to UserDtoInput ?
so, you can use this dto in the register, create and update cases